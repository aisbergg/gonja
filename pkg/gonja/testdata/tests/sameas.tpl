{{ simple is sameas simple }}  # map is not addressable
{{ simple is sameas(simple) }}  # same output, parentheses for arg
{{ 123456 is sameas simple }}
{{ 123456 is sameas 123456 }}
{{ number is sameas 11 }}
{{ simple.nil is sameas nil }}
{{ simple.nil is sameas None }}
{{ complex.user.Name is sameas complex.user.Name }}
