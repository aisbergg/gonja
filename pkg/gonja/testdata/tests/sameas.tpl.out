False  # map is not addressable
False  # same output, parentheses for arg
False
False
False
True
True
True
